examples/tf-netapp-cvs/main.tf

```diff
@@ -0,0 +1,173 @@
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+module "nfs-api" {
```

Comment:
Why do you use a module here, when you're using resources below?

Reply:
This module is being used to enable necessary APIs needed by NetApp CVS. https://registry.terraform.io/modules/terraform-google-modules/project-factory/google/latest?tab=inputs
The resources are specifically used to create and manage NetApp CVS volumes.

Reply:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_service

Reply:
Oh, got it. Thanks!
It's done.